src/recipes/jam-band.json

```
@@ -0,0 +1,53 @@
+{
+    "name": "<Jam Band>",
```

You can remove all the placeholder tokens in this file:
```suggestion
    "name": "Jam Band",
```

---

I believe I followed requested changes, thank you for reviewing.